### PR TITLE
Validations branch

### PR DESCRIPTION
**Pull request by nunomsantos**

### CODE REVIEW

File: app/models/attachment.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 class Attachment < ActiveRecord::Base
   mount_uploader :file_name, AvatarUploader
   belongs_to :space
+
```
Comment (nunoagomes) [inline app/models/attachment.rb]: Consider add 'validates :file_name, presence: true'

File: db/migrate/20150408151234_create_attachments.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 class CreateAttachments < ActiveRecord::Migration
   def change
     create_table :attachments do |t|
-      t.belongs_to :space, index: true
+      t.belongs_to :space, index: true, null: false
       t.string :file_name
```
Comment (nunoagomes) [inline db/migrate/20150408151234_create_attachments.rb]: Consider add 't.string :file_name, null: false'

**Comment by nunoagomes** [comment]
After these small changes that I mentioned feel free to merge your branch. Great effort and very nice work!!!

File: app/models/attachment.rb (hunk 1 above)
Comment (nunomsantos) [inline app/models/attachment.rb]: Thanks, I'll do it.

File: db/migrate/20150408151234_create_attachments.rb (hunk 2 above)
Comment (nunomsantos) [inline db/migrate/20150408151234_create_attachments.rb]: Nice catch. I'll change it.

**Comment by nunomsantos** [comment]
Thanks. I made all the changes. I'll merge after passing the tests.

**Pull request merged**